src/services/pcn-synflood/swagger_api/synflood_swagger

```diff
@@ -714,9 +712,8 @@
                         "description": "OK: Successful operation",
                         "schema": {
                             "readOnly": true,
-                            "type": "integer",
-                            "description": "pkts delivered to application over total received pkts",
-                            "format": "uint64"
+                            "type": "string",
+                            "description": "pkts delivered to application over total received pkts"
```

Comment:
Ratio between the number of IP pkts delivered to application protocols and the total number of received pkts